tooling/vector_dbs/cw-milvus/templates/backup_job.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "cw-milvus.name" . }}-script
+data:
+  backup.sh: |
+    {{ .Files.Get "scripts/backup.sh" | nindent 4 }}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: milvus-backup
+spec:
+  schedule: {{ .Values.backupCronExpression }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          initContainers:
+          - name: prepare-files
+            image: busybox:latest
+            command: ["sh", "-c", "mkdir -p /milvus-backup/script && cp /scripts/backup.sh /milvus-backup/script/ && chmod +x /milvus-backup/script/backup.sh && cp -r /etc/milvus-backup/configs /milvus-backup/"]
+            volumeMounts:
+            - name: shared-data
+              mountPath: /milvus-backup
+            - name: script-volume
+              mountPath: /scripts/backup.sh
+              subPath: backup.sh
+            - name: config-volume
+              mountPath: /etc/milvus-backup/configs/backup.yaml
+              subPath: backup.yaml
+          containers:
+          - name: backup-runner
+            image: alpine:latest
```

Comment:
better to use [milvus backup](https://hub.docker.com/r/milvusdb/milvus-backup) image here